### DISCUSSION TITLE
Jaxlib release v0.4.33 link on release page

### DISCUSSION THREAD

**Discussion by truncs** [jax-ml/jax#23952] [General] [answered]
This is blocking folks on Arch to install jax 0.4.33 or was it decided to not have jax and jaxlib has separate releases?

**Comment by dfm** [comment, marked as answer]
Thanks for the question! We have actually updated the JAX release process so that there aren't separate tags. The released JAX and jaxlib is always be cut from the same commit and the versions are required to match. Going forward the best bet will be to use the `jax-*` tags for both. Does that clear it up for you?

**Reply by hawkinsp** [reply]
To add to that: we used to allow a wider range of versions of jaxlib for any given `jax` release. To simplify our software development process, we now require that you're either using:
* `jax` and `jaxlib` cut from the same commit or release.
* `jax` from head, with the most recent preceding released `jaxlib`. We allow the latter so you can work on JAX Python code from head without building any C++ code.

Since we release `jax` and `jaxlib` together, there's now only one tag for both.
